std/assembly/rt/pure.ts

```diff
@@ -15,7 +15,7 @@ import { onincrement, ondecrement, onfree, onalloc } from "./rtrace";
 // B: buffered
 
 // @ts-ignore: decorator
-@inline const BUFFERED_MASK: u32 = 1 << (sizeof<u32>() * 8 - 1);
+@inline const BUFFERED_MASK: u32 = 1 << ((sizeof<u32>() * 8) - 1);
```

Comment:
this unnecessary. Multiply have bigger precedence than subtraction

Reply:
I know but the syntax confused my IDE.

<img width="553" alt="Screen Shot 2019-06-07 at 1 06 00 PM" src="https://user-images.githubusercontent.com/1483244/59121053-0b6d0480-8925-11e9-90ea-f8130d290894.png">